pkg/translator/jaeger/jaegerproto_to_traces_test.go

```diff
@@ -417,6 +417,15 @@ func TestSetInternalSpanStatus(t *testing.T) {
 			status:           errorStatus,
 			attrsModifiedLen: 1,
 		},
+		{
+			name: "do not set error if http.status_code >= 400 and error tag is set to false",
```

Comment:
I think the actual condition should be: we should not set any error tag if one is already present. The status code shouldn't matter. For instance, if we get a "200" but there's an "error=true" set by something else, we should respect that.

Reply:
I agree @jpkrohling that's what I wished for when I filed the issue.